tests/ragger/test_eip1559.py

```diff
@@ -0,0 +1,62 @@
+from web3 import Web3
+
+from client.client import EthAppClient
+import client.response_parser as ResponseParser
+from client.utils import recover_transaction
+
+from ragger.backend import BackendInterface
+from ragger.firmware import Firmware
+from ragger.navigator import Navigator, NavInsID
+
+from constants import ROOT_SNAPSHOT_PATH
+
+
+BIP32_PATH = "m/44'/60'/0'/0/0"
+NONCE = 1
+GAS_PRICE = 40000
+GAS_LIMIT = 10
+ADDR = bytes.fromhex("cccccccccccccccccccccccccccccccccccccccc")
+AMOUNT = 0.01
+CHAIN_ID = 1
+
+
+def test_sign_eip_1559(firmware: Firmware,
```

Comment:
EIP-1559 already has a test in test_sign.py, might be redundant.

Reply:
Removed